Sources/Quick/QuickSelectedTestSuiteBuilder.swift

```diff
@@ -0,0 +1,69 @@
+/**
+ Responsible for building a "Selected tests" suite. This corresponds to a single
+ spec, and all its examples.
+ */
+internal class QuickSelectedTestSuiteBuilder: QuickTestSuiteBuilder {
+
+    /**
+     The test spec class to run.
+     */
+    let testCaseClass: AnyClass!
+
+    /**
+     For Objective-C classes, returns the class name. For Swift classes without,
+     an explicit Objective-C name, returns a module-namespaced class name
+     (e.g., "FooTests.FooSpec").
+     */
+    var namespacedClassName: String {
```

Comment:
I'm considering renaming this to just `className`, given that classes with Objective-C names aren't actually namespaced. What do you think?